### ISSUE TITLE
False positive fatal parse error when multiple variables to create css property

### ISSUE THREAD

**Issue by stephenkoo** [sasstools/sass-lint#1086]
**Sass version**
sass-lint ver 1.10.2

**Issue**
When using 2 or more variables to make up a property name like `margin-top` from `#{$variable-called-margin}-#{$variable-called-top}`, a false fatal parse error occurs on [**line 53, 55 & 56 here**](https://www.sassmeister.com/gist/e23c2952ffbf42f0b13c1b42d3d7d449), even though it compiles correctly.

Error message: 'Error: Please check validity of the block starting from line ... (Fatal)'

![screen shot 2017-06-05 at 9 37 38 am](https://cloud.githubusercontent.com/assets/18624246/26766331/b0007386-49d2-11e7-97c4-a4810642fc37.png)

Code found in link above:
```scss
// ----
// Sass (v3.4.21)
// Compass (v1.0.3)
// ----

@mixin above($width) {
  @media only screen and (min-width: $width) {
    @content
  }
}

@mixin below($width) {
  @media only screen and (max-width: $width - 1) {
    @content
  }
}

// Each vertical-space-units row (v-space unit row) represents a v-space unit.
// Each column (v-space size) represents a v-space unit’s size at:
// (1) below sm, (2) above sm, (3) above md
$vertical-space-units: (
  (0.375rem, 0.5rem, 0.75rem),
  (  0.5rem,   1rem,  1.5rem),
  (  1.5rem,   2rem,    3rem),
  ( 2.25rem,   3rem,  4.5rem),
  (    3rem,   4rem,    6rem)
);

@mixin space-units($v-space-size) {
  @for $v-space-unit-number from 1 through length($vertical-space-units) {
    $v-space-unit: nth($vertical-space-units, $v-space-unit-number);
    $space-types: (m: margin, p: padding);
    $space-sides: (t: top, b: bottom, y: top bottom);
    
    @each $space-type-label, $space-type in $space-types {
      @each $space-side-label, $space-side in $space-sides {
        @include v-space-selector($space-type-label, $space-side-label, $v-space-unit-number) {
          @include v-space-declarations($space-type, $space-side, $v-space-unit, $v-space-size)
        }
      }
    }
  }
}

@mixin v-space-selector($space-type-label, $space-side-label, $v-space-unit-number) {
  .#{$space-type-label}#{$space-side-label}-#{$v-space-unit-number} {
    @content
  }
}

@mixin v-space-declarations($space-type, $space-side, $v-space-unit, $v-space-size) {
  @if type-of($space-side) != "list" {
    #{$space-type}-#{$space-side}: nth($v-space-unit, $v-space-size);
  } @else {
    #{$space-type}-#{nth($space-side, 1)}: nth($v-space-unit, $v-space-size);
    #{$space-type}-#{nth($space-side, 2)}: nth($v-space-unit, $v-space-size);
  }
}

@include space-units(1);
@include above(sm) { @include space-units(2) };
@include above(md) { @include space-units(3) };
```
**Why I think creating a property name using multiple sass variables is the cause of parse error**
When renaming either `#{$space-type}` or `#{$space-side}` to a non-variable text like `#{$space-type}-top:` or `margin-#{$space-side}`, the error disappears.

**Comment by bgriffith** [comment]
This is fixed in the newly released version `v1.11.0`. Please let me know if you still have issues. Thanks!
